### DISCUSSION TITLE
scripts.search VS scripts.train

### DISCUSSION THREAD

**Discussion by imruljubair** [Project-MONAI/tutorials#1636] [General]
Hi,

I am new in Medical Image Segmentation and with MONAI. I am really interested to apply Auto3Dseg for my project. I am confused what is the difference between `scripts.search` and `scripts.train`. I observed that `scripts.search` also does training.

Looking forward.

Jubair

**Comment by KumoLiu** [comment]
Hi @imruljubair, it's dints-specific. For more information, please refer to the parper.
https://arxiv.org/abs/2103.15954
Thanks.
